lib/liftoff/xcodeproj_helper.rb

```diff
@@ -78,11 +78,23 @@ def save
     private
 
     def target
-      @target ||= ObjectPicker.choose_item('target', available_targets)
+      @target ||= ObjectPicker.choose_item('target', application_targets)
+    end
+
+    def test_target
+      @test_target ||= ObjectPicker.choose_item('test target', test_targets)
+    end
+
+    def all_targets
```

Comment:
Should this be lower down in the file, so that higher-level methods (that use this one) are above it? I like to go higher-to-lower as I read down a file.